packages/navigator/src/browser/navigator-widget.tsx

```diff
@@ -194,4 +233,20 @@ export class FileNavigatorWidget extends FileTreeWidget {
         </div>;
     }
 
+    protected createWorkspaceRoot(workspace: FileStat, children: TreeNode[]): DirNode {
+        const id = workspace.uri;
+        const uri = new URI(id);
+        return {
+            id,
+            uri,
+            fileStat: { ...workspace, isDirectory: true },
```

Comment:
Should we really reuse `FileStatNode` for `WorkspaceNode`. What for? it is not a file. How about defining a new type?